src/main/kotlin/io/gitlab/arturbosch/detekt/sonar/sensor/DetektConfiguration.kt

```diff
@@ -44,6 +46,11 @@ internal fun createSpec(baseDir: Path, configuration: Configuration): Processing
     }
 }
 
+internal fun getInputPaths(configuration: Configuration, basePath: Path): List<Path> =
+    configuration.get(PATH_INPUT_DIRS_KEY).map { sources ->
+        sources.split(",").map { Path.of(it.trim()) }
+    }.orElse(listOf(basePath))
```

Comment:
I'm wondering if instead of using `.orElse` we should merge the two sources?

Reply:
No! Just the current problem is that InputPaths contains baseDir. This leads to the fact that the detector starts to process caches/acceessor.